vllm/core/block/prefix_caching_block.py

```diff
@@ -811,6 +826,10 @@ def token_ids(self) -> List[int]:
     def prev_block(self) -> Optional[Block]:
         return self._prev_block
 
+    @property
+    def extra_hash(self) -> Optional[int]:
+        return self._extra_hash
+
```

Comment:
Do you really need to expose this property?

Reply:
I believe this is necessary because the block's `_extra_hash` should be referenced in `PrefixCachingBlockAllocator`, just like other properties such as `_token_ids` and `_prev_block`.

e.g.)
```
tmp_block = self.allocate_immutable_block(
   prev_block=block.prev_block,
   token_ids=block.token_ids,
   extra_hash=block.extra_hash)
```